src/it/test-projects/creedengo-java-plugin-test-project/src/main/java/org/greencodeinitiative/creedengo/java/checks/AvoidRuntimeExceptions.java

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
I think a best name is "AvoidCatchingRuntimeExceptions" (apply this feedback to all files)